client/src/components/Workbench/Workbench.jsx

```diff
@@ -54,6 +56,7 @@ function Workbench() {
     });
 
     if (setupComplete) {
+      fetchSystems();
```

Comment:
This is also called one level up in `client/src/components/Workbench/AppRouter.jsx` -- do we need this call as well?

Reply:
I'll take a look, this is probably a call that was duplicated in the code before I refactored.